addons/website/static/src/snippets/s_tabs/001.scss

```diff
@@ -2,24 +2,38 @@
 .s_tabs[data-vcss="001"] {
     .s_tabs_content {
         &.s_tabs_slide_up, &.s_tabs_slide_down, &.s_tabs_slide_left, &.s_tabs_slide_right {
-            > .tab-pane.fade {
-                transition: all 0.2s;
-            }
-            > .tab-pane.fade.show {
-                transform: translateX(0rem) translateY(0rem);
+            > .tab-pane {
+                animation-duration: 0.2s;
             }
         }
-        &.s_tabs_slide_up > .tab-pane.fade {
-            transform: translateY(-1rem);
+        &.s_tabs_slide_up > .tab-pane {
+            animation-name: slide_up;
         }
-        &.s_tabs_slide_down > .tab-pane.fade {
-            transform: translateY(1rem);
+        &.s_tabs_slide_down > .tab-pane {
+            animation-name: slide_down;
         }
-        &.s_tabs_slide_left > .tab-pane.fade {
-            transform: translateX(-1rem);
+        &.s_tabs_slide_left > .tab-pane {
+            animation-name: slide_left;
         }
-        &.s_tabs_slide_right > .tab-pane.fade {
-            transform: translateX(1rem);
+        &.s_tabs_slide_right > .tab-pane {
+            animation-name: slide_right;
         }
     }
+
+    @keyframes slide_up {
+        from {transform: translateY(-1rem);}
+        to {transform: translateX(0rem) translateY(0rem);}
+    }
+    @keyframes slide_down {
+        from {transform: translateY(1rem);}
+        to {transform: translateX(0rem) translateY(0rem);}
+    }
+    @keyframes slide_left {
+        from {transform: translateX(-1rem);}
+        to {transform: translateX(0rem) translateY(0rem);}
+    }
+    @keyframes slide_right {
+        from {transform: translateX(1rem);}
+        to {transform: translateX(0rem) translateY(0rem);}
+    }
```

Comment:
@sbuhl Are you sure you want a preview for the slide animation of tabs (actually... do we really want that option? Feels useless to me, who cares about that?)

Reply:
@qsm-odoo @bvr-odoo 
You can remove the preview but keep the transitions. It's like an animate.